app/validators/blocks/block_validator.py

```diff
@@ -3,12 +3,16 @@
 
 
 class BlockValidator(Validator):
-    METADATA_REFERENCE_INVALID = "Invalid metadata reference"
-    ANSWER_REFERENCE_INVALID = "Invalid answer reference"
-    LIST_REFERENCE_INVALID = "Invalid list reference"
-    ANSWER_SELF_REFERENCE = "Invalid answer reference (self-reference)"
     ACTION_PARAMS_MISSING = "Action params key missing"
     ACTION_PARAMS_SHOULDNT_EXIST = "Action params key should not exist"
+    ANSWER_REFERENCE_INVALID = "Invalid answer reference"
+    ANSWER_SELF_REFERENCE = "Invalid answer reference (self-reference)"
+    COMPOSITE_ANSWER_INVALID = "Invalid composite answer"
+    COMPOSITE_ANSWER_FIELD_INVALID = "Invalid field for composite answer"
+    LIST_REFERENCE_INVALID = "Invalid list reference"
+    METADATA_REFERENCE_INVALID = "Invalid metadata reference"
+
+    COMPOSITE_ANSWERS = {"Address": ["line1", "line2", "town", "postcode"]}
```

Comment:
Name implies this is a list etc maybe `COMPOSITE_ANSWERS_TO_SELECTORS_MAP` to be explicit?